index.bs

```diff
@@ -1499,7 +1500,10 @@ input parameters:
 - A list of credentials acceptable to the [=[RP]=] (possibly filtered by the client).
 - Extension data created by the client based on the extensions requested by the [=[RP]=].
 
-When this method is invoked, the [=authenticator=] must perform the following procedure:
+The [=authenticator=] must perform the following procedure depending on the PublicKeyCredentialType of the related [=credential=]:
```

Comment:
Basically, you need to walk through the steps you're asking folks to implement, one by one, and make sure that they're what you intend and that each step only uses information that previous steps have already said how to compute. I can't write the spec for you or give you a precise recipe for how to fix it because I don't know what you intend to specify.